contrib/graphite/blueflood.py

```diff
@@ -43,37 +77,133 @@ def __init__(self, config=None):
 
     self.tenant = tenant
     self.bf_query_endpoint = urls[0]
+    self.enable_submetrics = enable_submetrics
+    self.submetric_aliases = submetric_aliases
+    self.client = BluefloodClient(self.bf_query_endpoint, self.tenant, 
+                                  self.enable_submetrics, self.submetric_aliases)
+    print("BF finder submetrics enabled: ", enable_submetrics)
+
+  def complete(self, metric, query_depth):
+    metric_parts = metric.split('.')
+    if query_depth < len(metric_parts):
```

Comment:
Totally up to you, but what do you think about consolidating these if statements into just one block like, `if (query_depth < len(metric_parts)) or (self.enable_submetrics and (query_depth == len(metric_parts)))`